.clang-format

```diff
@@ -0,0 +1,45 @@
+# -*- yaml -*-
```

Comment:
Can you merge with master? I think this is a duplicate addition.

Reply:
Done